tests/test_esm_runscripts/test_filedicts.py

```diff
@@ -12,12 +12,20 @@
   unit test, which you don't really want.  
 """
 import os
+from pathlib import Path
 
 import yaml
-
-from pathlib import Path
+import pytest
 
 import esm_runscripts.filedicts
+from esm_runscripts.filedicts import resolve_file_movements
+
+
+@pytest.fixture()
+def config():
+    """Generates fake config to be used before each test"""
+    fake_config = dict()
+    yield fake_config
```

Comment:
Why does this need to be a yield? (Just out of curiosity)

Reply:
`yield` is recommended by pytest: https://docs.pytest.org/en/7.1.x/how-to/fixtures.html#yield-fixtures-recommended